erpnext/stock/doctype/item/item.json

```diff
@@ -1115,14 +1117,24 @@
    "fieldtype": "Table",
    "label": "Shipping Information",
    "options": "Shipping Information"
+  },
+  {
+   "fieldname": "flowkana_details_section",
+   "fieldtype": "Section Break",
+   "label": "Flowkana Details"
+  },
+  {
+   "fieldname": "ivt_id",
```

Comment:
```suggestion
   "fieldname": "inventory_id",
```

Reply:
@hrwX Flowkana uses ivt_id and I wanted to keep it consistent - is that okay?